### PR TITLE
Display validation results

### PR DESCRIPTION
**Pull request by ageorgou**
Addresses #29. Have commented out the communication with the server temporarily so we can use the mock result.

This
- shows a popup with the overall result of the validation
- logs the server response (to console and file)
- underlines the lines with errors

### CODE REVIEW

**Comment by giordano** [comment]
Slightly related to this pull request, in https://github.com/oracc/nisaba/blob/beb2642cc9b5a5a9d8b8553b4fa74fbdb1b2b1e9/src/client/server_result.ts#L77 can we write instead
```ts
`${real-file-path}:${line_num}: ${error_msg}.`
```
(as long as we can get the real path of the local ATF in a sane way)?  In this way the `file:line` will be clickable, matching the behaviour in Nammu.

**Comment by giordano** [comment]
Clicking on the validate button multiple times I get this error message:

image

I don't see the failure the first time I click on it the first time in a session.  I'm trying to figure out what's the problem, but to no avail so far

**Review by giordano — APPROVED** [review]
Looks great!  Only question: should we keep mock results for the moment and replace with real ones later?

**Comment by ageorgou** [comment]
I've left the mock result in so that it at least does something! I was planning to work on getting the real ones soon, but I can add that part back in (uncomment the server communication).

File: src/extension.ts [hunk 1]
```diff
@@ -30,12 +32,13 @@ export function activate(context: vscode.ExtensionContext) {
     context.subscriptions.push(disposable1);
 
     const disposable2 = vscode.commands.registerCommand('ucl-rsdg.validateAtf', () => {
-        // The code you place here will be executed every time your command is 
-        const filePath = vscode.window.activeTextEditor.document.uri.fsPath;
+        const editor = vscode.window.activeTextEditor;
+        const filePath = editor.document.uri.fsPath;
         const fileProject = "cams/gkab";
```
[the remark this answers is not in the document]
Comment (ageorgou) [reply src/extension.ts:37]: Yeah, that can change when we finalise the request (#30)...

**Pull request merged**